src/project/qml/MuseScore/Project/internal/ChooseInstrumentsAndTemplatesPage.qml

```diff
@@ -151,5 +153,9 @@ Item {
             id: templatePage
             navigationSection: root.navigationSection
         }
+
+        Component.onCompleted: {
+            templatePage.done.connect(root.done)
+        }
```

Comment:
It seems this bit can be simplified, by using the `onDone:` signal handler (which is automatically generated from the `done` signal in ChooseTemplatePage.qml):
```suggestion
            id: templatePage
            navigationSection: root.navigationSection
            
            onDone: {
                root.done()
            }
        }
```